cmd/minikube/cmd/service.go

```diff
@@ -77,6 +77,10 @@ var serviceCmd = &cobra.Command{
 
 		urls, err := service.WaitForService(api, namespace, svc, serviceURLTemplate, serviceURLMode, https, wait, interval)
```

Comment:
because getting list of ALL services is a very quick call, 
First, get a list of services and then check if the service is in that list , if there isnt return quick,
then Seconly if the service exist, then wait for the service to be up !